util/prefixes.go

```diff
@@ -17,3 +17,17 @@ func PrefixRegistryToImageName(registry string, img string) string {
 
 	return img
 }
+
+// PrefixTags prefixes tags in the specified command and returns the new command.
+func PrefixTags(cmd string, registry string) string {
+	if registry == "" {
+		return cmd
+	}
+	fields := strings.Fields(cmd)
+	for i := 1; i < len(fields); i++ {
+		if fields[i-1] == "-t" || fields[i-1] == "--tag" {
+			fields[i] = PrefixRegistryToImageName(registry, fields[i])
```

Comment:
You might be able to return tags array from this function as well so you don't need to parse rendered string twice.

Reply:
Yeah, definitely. I'm gonna add this to #243 and merge this for now so that you're unblock at least.